modules/microshift-enabling-workload-partitioning.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-workload-partitioning.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-enabling-workload-partitioning_{context}"]
+= Enabling workload partitioning
+
+To enable workload partiotioning on {microshift-short},make the following configuration changes:
+
+* Update and embed the Kubelet configuration file in the {microshift-short} `config.yaml` file to enable and configure CPU Manager for the workloads.
+* Update the CRI-O configuration file with the cpuset and management annotation.
+* Update the systemd configuration file with CPUAffinity setting for the {microshift-short} and CRI-O services.
+
+.Procedure
+
+. Update and embed the Kubelet configuration in the {microshift-short} `config.yaml` file:
+* Update the Kubelet configuration file in the path `/etc/kubernetes/openshift-workload-pinning` with the `cpuset` value. The Kubelet configuration instructs the kubelet to modify the node resources with the capacity and allocatable CPUs.
++
+.Example of Kubelet configuration
+[source,yaml]
+----
+# ...
+{
+  "management": {
+    "cpuset": "0,6,7"
+  }
+}
+# ...
+----
+* Update the {microshift-short} config.yaml file in the path `/etc/microshift/config.yaml`.Embed the Kubelet configuration in the {microshift-short} `config.yaml` file to enable and configure CPU Manager for the workloads.
++
+.Example of {microshift-short} `config.yaml` file
+[source,yaml]
+----
+# ...
+kubelet:
+  reservedSystemCPUs: 0,6,7<1>
+  cpuManagerPolicy: static
+  cpuManagerPolicyOptions:
+    full-pcpus-only: "true"<2>
+  cpuManagerReconcilePeriod: 5s
+# ...
+----
+<1> exclusive cpuset for the system/microshift daemons and the interrupts/timers.
```

Comment:
this  for  kubelet system daemon its not directly affecting Microshift daemon. we have another configurations for Microshift (systemd  unit configuration).

Reply:
done